src/Surfnet/ServiceProviderDashboard/Domain/Repository/TypeOfServiceRepositoryFromConfig.php

```diff
@@ -59,7 +59,8 @@ private function load(): void
         }
         $this->collection = new TypeOfServiceCollection();
         foreach ($data as $entry) {
-            $typeOfService = new TypeOfService($entry->typeEn, $entry->typeNl);
+            $typeHidden = $entry?->typeHidden ?? false;
```

Comment:
Nitpick: My opinion is that the use of the null coalescing operator `??` combined with the null safe operator `?->` makes this line somewhat hard to understand at once. Consider rewriting this line into maybe an if-statment?

Might be my lesser experience in using the 'modern' operators :sauropod: . And feel free to leave as is.

Reply:
:t-rex: 
```suggestion
            $typeHidden = $entry->typeHidden ?? false;
```

Reply:
Good sense of smell, it was redundant. Fixed

Reply:
Much better like this